contents/handbook/growth/marketing/index.md

```diff
@@ -1,335 +1,134 @@
+**PostHog Cloud** is positioned to service clients who need less control.
 
-With our marketing team expanding, we're upping the stakes and setting a goal of getting 90%+ of startups in new YC batches and 75% of startups from the last 2 years that have not chosen a product analytics solution, to use PostHog.
+**PostHog Self Hosted** is positioned to service clients who need more control.
 
-We're betting on YC startups because YC acts as strict screening process for successful startups and founders. By supporting these startups, we're aligning ourselves with many future successful companies and expanding our reach and awareness to their rapidly growing network and even further within the wider YC and Hacker News network. We're betting that this will be significantly more effective than any other traditional content or paid strategies.
 
-To incentivise YC startups to choose PostHog for the product analytics, we'll offer:
+## Target audience
 
-- Generously increased usage on PostHog Cloud
-- Extended free premium licenses for on-premise
-- Free VPC setup
-- Free support hours
-- Free onboarding, training and advice calls to get your demo day ready
-- Additional free licenses, hardware etc to be determined
-- Access to our growing library of tutorials and other resources in our docs
+*Innovative technical teams*, more commonly found in startups, who care about:
 
-### The best developer community
+- a unified product analytics platform
+- open source
+- control: hosting, pricing, source, data, privacy and security
 
-We want to build the most active and supportive developer community around, by supporting and celebrating our community's contributions more than any other community.
+We will expand to non-technical teams when we have achieved technical awareness saturation.
 
-We'll create resources to make it easier to contribute to different areas of PostHog, whatever your level of experience.
 
-We'll create tools and apps that celebrate our community's growth and members' contributions.
+## Goals
 
-We'll thank substantial contributions with swag and other sponsorships.
+### Goal 1: User acquisition
 
-We'll financially support community lead events.
+Increase:
 
-We'll make it easy to contribute to in-the-monent and long-running discussions using open tools.
+- Open source self hosted installs
+- Premium self hosted leads
+- Free cloud signups
+- Premium cloud signups
 
-### Enterprise early adopters
+### Goal 2: Community growth
 
-We'll forge deep relationships with our early enterprise customers who will be critical to develop our early enterprise product marketing and reaching more traditional enterprise engineering teams.
+Increase:
 
-We're interested to learn about their processes, their awareness and adoption of product analytics solutions, enterprise specific solutions, pricing and technical and compliance requirements.
+- GitHub stars
```

Comment:
Have you thought about the role marketing may play in growing community-led plugins?

Reply:
I think marketing will play a bit role. At the moment that's capture by overall contributions but I have discussed adding a separate metric for plugins to focus efforts.

Reply:
Added a specific metric for this.